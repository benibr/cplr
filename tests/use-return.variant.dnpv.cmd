../cplr 'ret = 69' -dnpv
