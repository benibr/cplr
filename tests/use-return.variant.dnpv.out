Preparation phase
Generation phase
Generating code
================================================================================
     1	/* main */
     2	int main(int argc, char **argv) {
     3		int ret = 0;
     4	/* statements */
     5	 s0:	ret = 69;
     6	/* done */
     7		return ret;
     8	}
================================================================================
Generated: 182 bytes code, 117 bytes dump
Compilation phase
Compilation succeeded
